opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -65,25 +67,34 @@
     Span setTag(String key, Number value);
 
     /**
-     * Add a new log event to the Span, accepting an event name string and an optional structured payload argument.
-     *
-     * If specified, the payload argument may be of any type and arbitrary size, though implementations are not
-     * required to retain all payload arguments (or even all parts of all payload arguments).
-     *
-     * The timestamp of this log event is the current time.
-     **/
-    Span log(String eventName, /* @Nullable */ Object payload);
+     * Log key:value pairs to the Span with the current walltime timestamp.
+     *
+     * <p>A contrived example (using Guava, which is not required):
+     * <pre>{@code
+     span.log(
+         ImmutableMap.Builder<String, Object>()
```

Comment:
I was imagining that application code callers would already have Guava or something similar on hand...